### DISCUSSION TITLE
Resize della card picture in lovelace

### DISCUSSION THREAD

**Discussion by vinny74man** [indomus/forum#1862] [HUB: Home Assistant]
Ciao a tutti scusate sarebbe possibile modificare la dimensione di una card picture in lovelace, sto facendo una nuova plancia e in una pila orizzontale devo inserire diversi elementi tra i quali un'immagine, il problema è che la card picture ha una dimensione minima più grande delle altre card, e quindi vorrei restringerla ma non sono riuscito a trovare il modo, qualcuno può aiutarmi?

Grazie

**Comment by 50494554524F** [comment]
in hacs trovi lovelace-layout e card-mod con i quali puoi fare di tutto...

**Comment by vinny74man** [comment]
Grazie ho provato lovelace-layout e card-mod e con il secondo sono riuscito ad ottenere quello che volevo.
Grazie
